config/main.py

```diff
@@ -3534,6 +3534,34 @@ def delete(ctx):
 
     sflow_tbl['global'].pop('agent_id')
     config_db.set_entry('SFLOW', 'global', sflow_tbl['global'])
+@config.group(cls=AbbreviationGroup)
+@click.pass_context
+def switch(ctx):
+    """switch related config"""
+    pass
+
+@switch.command('shell')
+@click.pass_context
+@click.argument('en', metavar='<en>', required=True, type=int)
+def set_sdk_diag_shell(ctx, en):
+    tmp_file = "/tmp/swss.json"
+    json_file = "/etc/swss/config.d/switch.json"
+    command = " docker cp   swss:" + json_file + " " + tmp_file
+
+    subprocess.Popen(command, shell=True)
+    with open (tmp_file) as f:
+        cfg = json.load(f)
+
+    cfg[0]["SWITCH_TABLE:switch"]["sdk_diag_shell"] = en
```

Comment:
can you handle try catch if switch.json doesn't have "sdk_diag_shell"?

Reply:
swssconfig will handle unsupported attributes